packages/editor/src/components/post-template/classic-theme.js

```diff
@@ -177,8 +177,7 @@ function PostTemplateDropdownContent( { onClose } ) {
 			{ canEdit && onNavigateToEntityRecord && (
 				<p>
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
Activate a classic theme (e.g. Twenty Twelve) and open a page in the block editor.

<img src="https://github.com/user-attachments/assets/6ad63cdd-ba60-4a7f-af64-0ef0a25c2a60" alt="Page template popover" width="294">

No visible change because this is a `link` variant.